independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/InboundSseEventImpl.java

```diff
@@ -122,12 +123,16 @@ public <T> T readData(GenericType<T> type, MediaType mediaType) {
         try {
             return (T) ClientSerialisers.invokeClientReader(null, type.getRawType(), type.getType(),
                     mediaType, null, null, new QuarkusMultivaluedHashMap<>(),
-                    serialisers, in, Serialisers.NO_READER_INTERCEPTOR, configuration);
+                    serialisers, in, getReaderInterceptors(), configuration);
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
     }
 
+    public ReaderInterceptor[] getReaderInterceptors() {
+        return configuration.getReaderInterceptors().toArray(Serialisers.NO_READER_INTERCEPTOR);
```

Comment:
Nitpicking but my understanding is that you will call `readData` a lot but this will be somewhat immutable? So maybe let's store it to avoid creating a new array each time?

Reply:
Yeah, that makes sense